python/ngraph_bridge/__init__.in.py

```diff
@@ -201,12 +201,20 @@ def is_grappler_enabled():
 
 def update_config(config, backend_name = "CPU", device_id = ""):
     #updating session config if grappler is enabled
+    opt_name = 'ngraph-optimizer'
```

Comment:
should the opt_name be defined inside the if

Reply:
moved inside